shell/platform/embedder/embedder.h

```diff
@@ -31,6 +31,119 @@ typedef enum {
   kSoftware,
 } FlutterRendererType;
 
+typedef enum {
```

Comment:
Can there be some reuse with https://github.com/flutter/engine/blob/3978f07530410b45e1aa972eb2bf758d032af71a/lib/ui/window/window.h#L31 to avoid that we have to maintain this in yet another location?